cl/lib/model_helpers.py

```diff
@@ -5,10 +5,10 @@
 def make_upload_path(instance, filename):
     """Return a string like pdf/2010/08/13/foo_v._var.pdf"""
     try:
-        # Cannot do proper type checking here because of circular import
-        # problems when importing Audio, Document, etc.
-        d = instance.file_with_date
```

Comment:
Makes sense. I'll look into that. The kind of transient `file_with_date` property thing seemed mysterious as I didn't see it defined on any models.